tests/server/schemas/test_core.py

```diff
@@ -325,9 +325,15 @@ async def test_valid_work_pool_default_queue_id(self):
         "template",
         [
             {
-                "job_configuration": {"thing_one": "{{ expected_variable }}"},
+                "job_configuration": {
+                    "thing_one": "{{ expected_variable_1 }}",
+                    "thing_two": "{{ expected_variable_2 }}",
+                },
                 "variables": {
-                    "properties": {"wrong_variable": {}},
+                    "properties": {
+                        "not_expected_variable_1": {},
+                        "expected_variable_2": {},
+                    },
```

Comment:
Can you add a new `template` value to this parameterized test so that we can keep the existing test case?

Reply:
sure, given it is very quick test that sounds like a good call to me. Will update accordingly in a bit

Reply:
should be now updated - the commit was made as a fixup